src/components/ec-panel/ec-panel.vue

```diff
@@ -78,65 +106,77 @@ export default {
 };
 </script>
 
-<style lang="scss">
-@import '../../scss/settings/index';
-@import '../../scss/tools/index';
-
-$ec-panel-header-btn-fill: $level-4-interactive-elements !default;
-$ec-panel-header-btn-fill-hover: $level-4-tech-blue !default;
-$ec-panel-background-color: $white !default;
+<style>
+@import '../../styles/tools/scrollbars.css';
+@import '../../styles/tools/transitions.css';
 
 .ec-panel {
-  width: 100%;
-  max-width: $side-panel-max-width;
-  height: 100vh;
-  min-height: 100%;
-  position: absolute;
-  background: $ec-panel-background-color;
-  right: 0;
-  top: 0;
-  padding: 16px 24px 24px 24px;
-  overflow-y: auto;
-
-  @include scrollbar;
-  @include left-box-shadow-level-1;
+  max-width: var(--ec-side-panel-max-width);
 
-  &__header {
-    display: block;
+  @apply tw-w-full;
+  @apply tw-absolute tw-right-0 tw-top-0;
+
+  &__content-container {
+    max-width: var(--ec-side-panel-max-width);
+
+    @apply tw-w-full tw-h-screen;
+    @apply tw-fixed;
+    @apply tw-flex tw-flex-col;
+    @apply tw-bg-light;
+    @apply tw-shadow-level-1-rtl;
   }
 
   &__header-icons {
-    display: flex;
-    flex-direction: row;
+    @apply tw-flex tw-flex-row;
   }
 
   &__header-icon {
-    fill: currentColor;
-    display: inline-block;
-    vertical-align: top;
+    @apply tw-fill-current;
+    @apply inline-block;
+    @apply tw-align-top;
   }
 
   &__header-action {
-    cursor: pointer;
-    color: $ec-panel-header-btn-fill;
+    @apply tw-cursor-pointer;
+    @apply tw-text-gray-4;
+    @apply tw-bg-transparent;
+    @apply tw-border-none;
 
-    @include color-transition;
+    @mixin ec-text-color-transition;
 
     &:hover {
-      color: $ec-panel-header-btn-fill-hover;
+      @apply tw-text-key-4;
     }
 
     &--back {
-      margin-right: auto;
+      @apply tw-mr-auto;
     }
 
     &--close {
-      margin-left: auto;
+      @apply tw-ml-auto;
     }
   }
+
+  &__header {
+    @apply tw-block;
+    @apply tw-mt-16 tw-mx-24;
+  }
+
+  &__main {
+    @apply tw-overflow-y-auto;
+    @apply tw-my-24;
+    @apply tw-ml-24;
+    @apply tw-pr-16;
```

Comment:
Why mixing padding and margin? Why width is different in both? It has nothing to do with the scrollbar, see screenshots.

<img width="354" alt="Screenshot 2020-06-17 at 14 09 35" src="https://user-images.githubusercontent.com/36676/84902155-5a0abe00-b0a4-11ea-9d58-93a51fad5a3c.png">
<img width="354" alt="Screenshot 2020-06-17 at 14 09 52" src="https://user-images.githubusercontent.com/36676/84902168-5bd48180-b0a4-11ea-830c-b128e0d295a4.png">